src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,27 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.Table;
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
+@Table(name = "movie")
 public class Movie implements Cloneable {
```

Comment:
This class implements Cloneable, but it's not a common practice for entities. Entities should not be cloned because they represent database records. Cloning entities can lead to unexpected behavior and bugs. Consider removing the Cloneable interface.